site/pages/review-disruption/[disruptionId].page.tsx

```diff
@@ -699,29 +735,45 @@
 
                         <input type="hidden" name="disruptionId" value={disruption.disruptionId} />
 
-                        <button className="govuk-button mt-8" data-module="govuk-button">
-                            {canPublish || disruption.template
-                                ? disruption.template
-                                    ? "Create template"
-                                    : "Publish disruption"
-                                : "Send to review"}
-                        </button>
-                        <button
-                            className="govuk-button govuk-button--warning ml-5 mt-8"
-                            data-module="govuk-button"
-                            onClick={(e) => {
-                                e.preventDefault();
-                                deleteActionHandler("disruption", [
-                                    {
-                                        name: "id",
-                                        value: disruption.disruptionId,
-                                    },
-                                ]);
-                            }}
-                        >
-                            {disruption.template ? "Delete template" : "Delete disruption"}
-                        </button>
-                        {!disruption.template && (
+                        {isEditingAllowed ? (
+                            <>
+                                <button className="govuk-button mt-8" data-module="govuk-button">
+                                    {canPublish || disruption.template
+                                        ? disruption.template
+                                            ? "Create template"
+                                            : "Publish disruption"
+                                        : "Send to review"}
+                                </button>
+                                <button
+                                    className="govuk-button govuk-button--warning ml-5 mt-8"
+                                    data-module="govuk-button"
+                                    onClick={(e) => {
+                                        e.preventDefault();
+                                        deleteActionHandler("disruption", [
+                                            {
+                                                name: "id",
+                                                value: disruption.disruptionId,
+                                            },
+                                        ]);
+                                    }}
+                                >
+                                    {disruption.template ? "Delete template" : "Delete disruption"}
+                                </button>{" "}
+                            </>
+                        ) : (
+                            <Link
+                                role="button"
+                                href={redirect}
```

Comment:
changed to go to /dashboard instead. This was only in the case that an operator user manages to get to the /review-disruption page for an LTA-made disruption. 

Operators cannot get to that page via clicking around on the site, but there is a change they could alter the disruption detail URL to go to the review disruption page. In this case, rather than redirecting they will go to the dashboard instead